lib/standard/string.nit

```diff
@@ -1957,6 +2025,11 @@ redef class Int
 	# Returns a string describing error number
 	fun strerror: String do return strerror_ext.to_s
 
+	# Converts an `Int` to its `UnicodeChar` equivalent
+	#
+	# `self` is supposed to be a code point in unicode.
+	fun to_unicode_char: UnicodeChar is extern "intcp_to_u8char"
```

Comment:
I don't know for this, I was always suspicious of the `to_c` and `ascii` services and how they're used in Nit.
I should probably check at the other languages to see how they do to make a proper decision.
Until then, ok I'll rename it `to_code_point` but it does feel strange to me.